src/Symfony/Component/Messenger/Command/StatsCommand.php

```diff
@@ -27,6 +29,12 @@
 #[AsCommand(name: 'messenger:stats', description: 'Show the message count for one or more transports')]
 class StatsCommand extends Command
 {
+    public const OUTPUT_FORMAT_TEXT = 'text';
+
+    public const OUTPUT_FORMAT_JSON = 'json';
+
+    public const ALL_OUTPUT_FORMATS = [self::OUTPUT_FORMAT_TEXT, self::OUTPUT_FORMAT_JSON];
```

Comment:
I don't see the value of having consts here, let's remove them IMHO.

Reply:
Removed the constants